src/styles/CardStyle/_index.scss

```
@@ -0,0 +1,32 @@
+@use './variables';
+
+/* #region Reset */
+:root {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* #endregion */
+
+/* #region Body */
+body {
+  margin: 0 auto;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: variables.$main;
+}
+
+/* #endregion */
+
+/* #region Image Component */
+.img-imac img {
```

move it to card.scss
```suggestion
.card__img {
```